### ISSUE TITLE
Error: EMFILE, open './node_modules/react/package.json'

### ISSUE THREAD

**Issue by blackdynamo** [jmreidy/grunt-browserify#340]
TL;DR; Was working, then stopped working, with no apparent changes.

Not sure this is the right project to post the issue. Sometime yesterday afternoon, I started receiving the error: `Error: EMFILE, open './node_modules/react/package.json'` when using grunt-browserify. My code hasn't changed, but the build doesn't seem to work. I am running OSX Yosemite, which I understand has some EMFILE issues. I have read up some solutions that recommend changing the number of open files using ulimit. Unfortunately this feels like a hack to me considering the issue only just appeared using versions of packages that haven't changed in several months.

After some investigation, I have narrowed down that using the grunt-browserify option watch:true stops the error from happening. Is there any issue with running watch:true for single builds when I don't want the project to keep the process alive? Why would this all of a sudden start popping up, not only on my machine but multiple other machines in the office (all OSX).

Simple GIST example for your pleasure.
https://gist.github.com/blackdynamo/59497daa26c876eddc6c

**Comment by tleunen** [comment]
Answered on reactiflux, we cannot really do anything about it because it depends on the value of `ulimit -n` on Mac OS X.
Not sure why it's not happening with `watch: true` though. But the upstream `browserify` repo might be a better place for this issue since it should also happen using the CLI.
